deploy/crds/kieapp.crd.yaml

```diff
@@ -1913,6 +1913,31 @@ spec:
                                 type: string
                               type: array
                           type: object
+                        kafkaJbpm:
+                          description: KafkaJBPMObject kafka configuration to be used
+                            by the KieApp for JBPM Emitter
+                          properties:
+                            acks:
+                              description: Jbpm Emitter, the number of acknowledgments
+                                the producer requires the leader to have received
+                                before considering a request complete.
+                              type: integer
+                            bootstrapServers:
+                              description: Jbpm emitter, a comma separated list of
+                                host/port pairs to use for establishing the initial
+                                connection to the Kafka cluster for Jbpm Emitter
+                              type: string
+                            clientID:
+                              description: Jbpm Emitter, identifier to pass to the
+                                server when making requests
+                              type: string
+                            maxBlockMs:
+                              description: Jbpm emitter, number of milliseconds that
+                                indicates how long publish method will bloc for jbpm
+                                emitter
+                              format: int32
+                              type: integer
+                          type: object
```

Comment:
what about configuration of these system properties via Operator:

- date format of message: org.kie.jbpm.event.emitters.kafka.date_format: The timestamp format for the time field of the messages. The default value is yyyy-MM-dd’T’HH:mm:ss.SSSZ 
- mapping of topic name for processes org.kie.jbpm.event.emitters.kafka.topic.processes: The topic name for process event messages. The default value is jbpm-processes-events.
- mapping of topic name for casesorg.kie.jbpm.event.emitters.kafka.topic.cases: The topic name for process event messages. The default value is jbpm-cases-events.
- mapping of topic name for tasks org.kie.jbpm.event.emitters.kafka.topic.tasks: The topic name for process event messages. The default value is jbpm-processes-tasks.

Reply:
- date format of message: value of date format is readed from the env "org.kie.server.json.date_format" https://github.com/kiegroup/jbpm/blob/master/jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java#L71 isn't a good idea override
- mapping of topic name for processes/case/tasks are fixed https://github.com/kiegroup/jbpm/blob/master/jbpm-event-emitters/jbpm-event-emitters-kafka/src/main/java/org/jbpm/event/emitters/kafka/KafkaEventEmitter.java#L163 we could map in the jboss-kie-modules